### PR TITLE
DYN-5235-NotificationsCenter-Localization Part1

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose
Localization of NotificationsCenter part1.
I've added two js methods, one will change the NotificationsCenter title window (by default is "Notifications"), the second one will change the bottom button text (current the default is "Mark all as read"). After this changes once they are published in npm.org this methods can be called from Dynamo so we can localize this content.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Localization of NotificationsCenter part1.

### Reviewers

@QilongTang

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
This is a GIF showing that in Dev Tools after executing the js methods, the Notifications title and the Button text are updated.
NotificationsCenterLocalization

File: src/App.js [hunk 1]
```diff
@@ -74,9 +80,33 @@ function App() {
     }
   };
 
+  const setTitle = (titleText) => {
+    setAPIData( prevState => {
+      console.log( prevState.notificabottomButtonText);
```
Comment (avidit) [inline src/App.js:85]: can you remove this line?

Comment (RobertGlobant20) [reply src/App.js]: sorry about that, I forgot it.
here is the fix: https://github.com/DynamoDS/NotificationCenter/pull/30/commits/6ab30190d950908162dd3a8e67a0f0ab2a1ea4d4

**Comment by QilongTang** [comment]
You will need to increase the version

**Comment by RobertGlobant20** [comment]
> You will need to increase the version
@QilongTang I guess there should be a way to do it automatically since sometimes I forget about it but that could be an enhancement.
here you go: https://github.com/DynamoDS/NotificationCenter/pull/30/commits/9699a66ddb3e33d383387fd6d23da182f652e670

**Pull request merged**